src/runtime/node/timers/$cloudflare.ts

```diff
@@ -0,0 +1,52 @@
+import type timers from "node:timers";
+
+export {
+  _unrefActive,
+  active,
+  clearInterval,
+  clearTimeout,
+  enroll,
+  promises,
+  setInterval,
+  setTimeout,
+  unenroll,
+} from "./index";
+
+export {
+  setImmediateFallback as setImmediate,
+  clearImmediateFallback as clearImmediate,
+} from "./internal/set-immediate";
+
+import {
+  _unrefActive,
+  active,
+  clearInterval,
+  clearTimeout,
+  enroll,
+  promises,
+  setInterval,
+  setTimeout,
+  unenroll,
+} from "./index";
+
+import {
+  setImmediateFallback as setImmediate,
+  clearImmediateFallback as clearImmediate,
+} from "./internal/set-immediate";
+
+globalThis.setImmediate = setImmediate;
+globalThis.clearImmediate = clearImmediate;
```

Comment:
the challenge with this implementation is that this module is now side-effecty which is what we've been trying to avoid in all modules in  unenv except for the `inject` config here: https://github.com/unjs/unenv/blob/f1c0bb8f0feeb22537e15b7f443af352dfb94777/src/presets/cloudflare.ts#L62-L69

could you please remove these two lines and add two new entries to `inject` section of the cloudflare.ts preset?

Reply:
you can use `globalThis.console` or `process` as implemented today via `inject` as the pattern to follow. thank you